### PR TITLE
admin dashboard: user, role manager

### PR DESCRIPTION
**Pull request by tung3457**

### CODE REVIEW

**Comment by tung3457** [comment]
ready

**Comment by tung3457** [comment]
ready

File: app/Http/Controllers/Admin/RoleController.php [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $roles = Role::all();
+        return view('backend.roles.index', compact('roles'));
```
Comment (BunnyPi04) [inline app/Http/Controllers/Admin/RoleController.php]: thiếu dòng trắng trước return

File: app/Http/Middleware/Manager.php [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+     */
+    public function handle($request, Closure $next)
+    {
+        if (!Auth::check()) {
+            return redirect('users/login');
+        }else {
+            $user = Auth::user();
+            if ($user->hasRole('Manager')) {
+                return $next($request);
+            }else {
```
Comment (BunnyPi04) [inline app/Http/Middleware/Manager.php]: convention

**Comment by BunnyPi04** [comment]
commented

**Comment by tung3457** [comment]
ready

**Pull request merged**